llvm/utils/TableGen/X86DisassemblerTables.cpp

```diff
@@ -878,7 +880,10 @@ void DisassemblerTables::emitContextTable(raw_ostream &o, unsigned &i) const {
   for (unsigned index = 0; index < ATTR_max; ++index) {
     o.indent(i * 2);
 
-    if ((index & ATTR_EVEX) || (index & ATTR_VEX) || (index & ATTR_VEXL)) {
+    if ((index & ATTR_EVEX) && (index & ATTR_64BIT) && (index & ATTR_OPSIZE) &&
+        (index & ATTR_ADSIZE))
+      o << "IC_EVEX_OPSIZE_ADSIZE";
```

Comment:
This would cause `IC_EVEX_OPSIZE_ADSIZE` to be print `ATTR_max` times, which sounds incorrect.